### ISSUE TITLE
Allow initial deploy to "update" all VMs across all AZs in parallel

### ISSUE THREAD

**Issue by drnic** [cloudfoundry/bosh#1943]
Currently a deployment will create VMs in parallel, but the "update" step is done 1 AZ at a time (and subsequently restricted by canary/max_in_flight).

For an initial deployment, it can be desirable to allow VMs across AZs to start up (initial "update") in batches that span AZs. E.g. a 5-node zookeeper cluster would start up all 5 nodes across 3 azs.

**Comment by cppforlife** [comment]
related: https://github.com/cloudfoundry/bosh-notes/blob/master/proposals/max-in-flight-v2.md

**Comment by mfine30** [comment]
Closing in favor of the linked bosh-note above
